lib/mongo/client.rb

```diff
@@ -238,10 +239,12 @@ def hash
     #   in which reads on a mongos are retried.
     # @option options [ Symbol ] :replica_set The name of the replica set to
     #   connect to. Servers not in this replica set will be ignored.
-    # @option options [ true, false ] :retry_writes Retry writes once when
+    # @option options [ true | false ] :retry_reads Retry reads once
```

Comment:
The tutorial documentation above indicates that setting this to false will enable legacy retryable reads, which isn't obvious from the documentation here. I think we should add a sentence explaining that setting it to false will enable legacy retryable reads.

Reply:
OK, what language would you suggest?

Reply:
`:retry_reads If true, modern retryable writes are enabled (which is the default). If false, modern retryable writes are disabled and legacy retryable writes are enabled`

Reply:
Made the suggested change, fixed to refer to retryable reads rather than retryable writes.